bindings/nodejs/examples/exchange/1-create-account.ts

```diff
@@ -10,6 +10,7 @@ import { Wallet, WalletOptions, CoinType } from '@iota/sdk';
 // This example uses secrets in environment variables for simplicity which should not be done in production.
 require('dotenv').config({ path: '.env' });
 
+// In this example we will create a wallet from mnemonic and a new account.
```

Comment:
There is already a description, remove one of them?

Reply:
Same for all exchange examples

Reply:
omg,  I need some days off :sweat_smile: , fixed